src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocvariable/InputJavadocVariableInner.java

```diff
@@ -1,11 +1,9 @@
 /*
 JavadocVariable
-scope = (default)private
-excludeScope = (default)null
+accessModifiers = (default)private
```

Comment:
Isn't the default all access modifiers? This doesn't make sense. @romani do we have a ticket somewhere to add validation for this?

Reply:
Validation is already in place, we might have defect in it. It is possible.